app-sso/service-operators/storage.hbs.md

```diff
@@ -29,6 +29,9 @@ To configure Redis as authorization server storage, you must have the following
 <strong>Caution:</strong>
 AppSSO takes _secure-by-default_ approach and will not establish non-encrypted communication channels.
 The `AuthServer` resource will enter an error state should a non-encrypted connection be attempted.
+
+<strong>Note:</strong>
+mTLS is not supported.
```

Comment:
**[suggestion]** - in addition to this statement, we can also gently relay to the service operator how a vanilla Redis turns off mutual TLS by setting client authentication to `no` as described in here: https://redis.io/docs/management/security/encryption/#client-certificate-authentication

Reply:
Great idea.